MathLib/LinAlg/UnifiedMatrixSetters.cpp

```diff
@@ -83,25 +83,35 @@ double norm(PETScVector const& x)
 void setVector(PETScVector& v,
                std::initializer_list<double> values)
 {
-    (void) v; (void) values;
-    // TODO implement
-}
+    std::vector<double> const vals(values);
+    std::vector<PETScVector::IndexType> idcs(vals.size());
+    std::iota(idcs.begin(), idcs.end(), 0);
 
+    v.set(idcs, vals);
+}
 
 void setMatrix(PETScMatrix& m,
                PETScMatrix::IndexType const rows,
                PETScMatrix::IndexType const cols,
                std::initializer_list<double> values)
 {
-    (void) m; (void) rows; (void) cols; (void) values;
-    // TODO implement
+    m.setZero();
```

Comment:
The PETSc docu does not say much about that. Cf. http://www.mcs.anl.gov/petsc/petsc-current/docs/manualpages/Mat/MatZeroEntries.html.
So let's hope for the best. ^^